applications/dashboard/views/message/edit.php

```diff
@@ -26,6 +26,7 @@
         <li class="form-group">
             <div class="label-wrap">
             <?php echo $this->Form->label('Message', 'Content'); ?>
+            <?php echo wrap(t("This field partially supports HTML."), 'div', ['class' => 'info']); ?>
```

Comment:
Where's the locales repo translation for this?

Reply:
https://github.com/vanilla/locales/pull/286